docs/man/tutorials.rst

```diff
@@ -16,37 +16,38 @@ This tutorial is provided in the WEC-Sim code release in the ``$WECSIM/tutorials
 Device Geometry
 ^^^^^^^^^^^^^^^^^^^^^^^
 The RM3 two-body point absorber WEC has been characterized both numerically and experimentally as a result of the DOE-funded `Reference Model Project <http://energy.sandia.gov/rmp>`_. 
-The RM3 is a two-body point absorber consisting of a float and a reaction plate. Full-scale dimensions of the RM3 and its mass properties are shown below.
+The RM3 is a two-body point absorber consisting of a float and a reaction  plate. Full-scale dimensions and mass properties of the RM3 are shown below. 
+
 
 .. figure:: /_static/images/RM3_Geom.png
    :width: 300pt
    :align: center
 
-+-----------------------------------------------------------+
-|Float Full Scale Properties                                |
-+=========+======+=========+================================+
-|Direction|      |Mass     |Moment of                       |
-+         +CG (m)+(tonne)  +Inertia (kg-m^2)                +
-+---------+------+---------+----------+----------+----------+
-|    x    |  0   |         |20,907,301|0         |0         |
-+---------+------+         +----------+----------+----------+
-|    y    |  0   |727.01   |0         |21,306,091|0         |
-+---------+------+         +----------+----------+----------+
-|    z    |-0.72 |         |0         |0         |37,085,481|
-+---------+------+---------+----------+----------+----------+   
-
-+-----------------------------------------------------------+
-|Plate Full Scale Properties                                |
-+=========+======+=========+================================+
-|Direction|      |Mass     |Moment of                       |
-+         +CG (m)+(tonne)  +Inertia (kg-m^2)                +
-+---------+------+---------+----------+----------+----------+
-|    x    |  0   |         |94,419,615|0         |0         |
-+---------+------+         +----------+----------+----------+
-|    x    |  0   |878.30   |0         |94,407,091|0         |
-+---------+------+         +----------+----------+----------+
-|    x    |-21.29|         |0         |0         |28,542,225|
-+---------+------+---------+----------+----------+----------+ 
++-------+---------------+
+| Body  | Mass (tonne)  |
++=======+===============+
+| Float | 727.01        |
++-------+---------------+
+| Plate | 878.30        |
++-------+---------------+
+
++-------+-----------+------------------------+--------------------------------------+
+| Body  | Direction | Center of Gravity* (m) | Moment of Inertia Tensor (kg m^2)    |
++=======+===========+========================+============+============+============+
+|       |    x      |  0                     | 20,907,301 | 0          | 0          |
+|       +-----------+------------------------+------------+------------+------------+
+| Float |    y      |  0                     | 21,306,091 | 0          | 0          |
```

Comment:
Change '21,306,091' from Mxy to Myy location

Reply:
@akeeste good catch